src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php

```diff
@@ -47,25 +45,18 @@ public function __construct(array $validationGroups, GroupRepository $groupRepos
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $groupRepository = $this->groupRepository;
-
-        $builder
-            ->add(
-                'groups',
-                'sylius_entity_to_identifier',
-                array(
-                    'label' => 'sylius.form.action.customer_group',
-                    'property' => 'name',
-                    'class' => $groupRepository->getClassName(),
-                    'query_builder' => function () use ($groupRepository) {
-                        return $groupRepository->getFormQueryBuilder();
-                    },
-                    'constraints' => array(
-                        new NotBlank(),
-                        new Type(array('type' => 'numeric')),
-                    ),
-                )
-            );
+        $builder->add('groups', 'sylius_entity_to_identifier', array(
```

Comment:
According to [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments) previous method arguments formatting was more consistent.